code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -73,3 +73,5 @@ var/global/default_martial_art = new/datum/martial_art
 
 	var/datum/body_accessory/body_accessory = null
 	var/tail // Name of tail image in species effects icon file.
+
+	var/list/splinted_limbs = list() //limbs we know are splinted

```

Comment:
`removed` on limbs needs to remove it from the `splinted_list` of the owner - this will fix both GC issues and null references.